bin/p-s.js

```diff
@@ -55,7 +56,25 @@ runPackageScript({
 function getPSConfig() {
   var psConfigFilename = program.config ? resolve(process.cwd(), program.config) : findUp.sync('package-scripts.js')
   try {
-    return require(psConfigFilename)
+    // Require a module like babel-register
+    if (program.require) {
+      try {
+        require(program.require)
```

Comment:
But if you require('some-file/in/your-project.js') node would cry too? So I think the only problem would be resolving the relative paths because I you don't need to resolve absolute paths because their already absolute and path.resolve('/some/absolute/path.js') would return '/some/absolute/path.js' anyway. Or I am wrong at this?

The other option I could imagine is Module.prototype.require.call(process.cwd(), programm.require) how about this?